.gitignore

```diff
@@ -34,3 +34,7 @@ bin/
 
 # vertical pod autoscaler test output
 vertical-pod-autoscaler/e2e/v1*/workspace/
+
+# vertical pod autoscaler vendor directories
+vertical-pod-autoscaler/vendor/
+vertical-pod-autoscaler/e2e/vendor/
```

Comment:
Could these be moved to `vertical-pod-autoscaler/.gitignore`.
Having a change in the top level `.gitignore` means that a different set of approvers is needed.

I did add line 35 in a previous PR, but those can be moved out later.